.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Test Matlab bindings'
+
+on: [push,pull_request]
+jobs:
+  test-solverdummies:
+    name: Test Matlab solverdummmies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - release: latest
+        - release: R2021b
+        - release: R2021a
+        - release: R2020b
+        - release: R2020a
```

Comment:
Should we also test for the latest Matlab release [2022a](https://www.mathworks.com/help/matlab/release-notes.html)? Technically this is the latest release of Matlab and we could test on it. This is not a must though.

Reply:
This is covered with `release: latest` or do you mean that we should explicitly name it here?

Reply:
Okay, if it is covered in `release: latest` then it is fine.